plugin/java-testing/src/test/java/com/buschmais/jqassistant/plugin/junit/test/rule/AssertExample.java

```diff
@@ -0,0 +1,21 @@
+package com.buschmais.jqassistant.plugin.junit.test.rule;
```

Comment:
The package name should start with `com.buschmais.jqassistant.plugin.java_testing` to be in sync with the Maven artifact id.

Reply:
Done.